README.md

```diff
@@ -2,13 +2,13 @@
 
 [![Build Status](https://travis-ci.org/SuperuserLabs/thankful.svg?branch=master)](https://travis-ci.org/SuperuserLabs/thankful)
 
-_Support the creators of free culture according to your preferences. Be thankful._
+_Support creators of free culture the way you want. Be thankful._
 
-Thankful is an app that keeps track of the websites you visit, the videos you watch, the music you listen to (not yet), which applications you use (such as Thankful itself), and then tries to figure out who created it so that you can choose to easily support those you find deserving by directly sending them cryptocurrency.
+Thankful is an app that keeps track of the websites you visit, the videos you watch, the music you listen to (not yet), which applications you use (such as Thankful itself) and then tries to figure out who created it so that you can choose to easily support those you find deserving, by directly sending them cryptocurrency.
 
-This helps solve a fundamental issue on the internet, where most culture is created and consumed today, where creators often have an larger incentive to simply get your attention as long as possible rather than to try and provide as much value to you as possible. So by using Thankful, you incentivize people to not only like what they are doing, but to make you feel that it's **worth supporting**.
+This helps solve a fundamental issue on the internet, where most culture is created and consumed today, where creators often have a larger incentive to simply get your attention as long as possible rather than to try and provide you with something of value. By using Thankful, you incentivize creators to not only spend more time and effort on what they do best, but to make sure that the content they create is **worth supporting**.
```

Comment:
We can probably remove `"where most culture is created and consumed today"` and change what follows it to `"where ad-funded creators have an incentive to steal your attention as long as possible instead of providing you with as much value as they can"`.

Also change the sentence after that to `"By using Thankful, you incentivize creators to not spend more time and effort on creating **what you want**"`.